telegram/chat.py

```diff
@@ -189,6 +189,20 @@ def __init__(
         self.bot = bot
         self._id_attrs = (self.id,)
 
+    @property
+    def full_name(self) -> Optional[str]:
+        """
+        :obj:`str`: Convenience property. If :attr:`first_name` is not :obj:`None` gives,
+        :attr:`first_name` followed by (if available) :attr:`last_name`.
+
+        .. versionadded:: 13.2
+        """
+        if not self.first_name:
+            return None
```

Comment:
wait, so this will always be None for not user chats? I think the docstring should reflect this, I wouldn't know as a first time API user that first_name is user only. Im not sure why people would use this over .effective_user, since that syntax is way clearer, but okay

Reply:
yay